src/rgb-hsv.js

```diff
@@ -43,18 +43,19 @@ export function hsv2rgb(hsvH, hsvS, hsvV) {
 	const rgbF = hsvH / 60 - rgbI & 1 ? hsvH / 60 - rgbI : 1 - hsvH / 60 - rgbI;
```

Comment:
The inequation `hsvH / 60 >= floor(hsvH / 60)` is always true. It's not necessary to consider a negative value.